src/index.js

```diff
@@ -119,6 +119,27 @@ class RCTVoice {
       });
     });
   }
+
+  /**
+   * (Android) Get a list of the speech recognition engines available on the device
+   * */
+  getSpeechRecognitionServices() {
+    return new Promise((resolve, reject) => {
+      if (Platform.OS !== 'android') {
+        reject('Speech recognition services can be queried for only on Android');
+        return;
+      }
+
+      Voice.getSpeechRecognitionServices((services, error) => {
```

Comment:
better make this returns a promise

Reply:
@sibelius I understand the need for docs and an example. You did mean to make the JS side return a Promise, not the Android side, right?

Reply:
the android side as well

check this https://github.com/wenkesj/react-native-voice/issues/133

and this https://github.com/wenkesj/react-native-voice/commit/7bc015afb4197ef59468cda94424457dc2a3424b

I want to unify all the apis to return promises instead of callbacks

Reply:
I wasn't aware that changes were being made to make all the methods use Promises instead of callbacks. Hence, I thought it'd be weird if only one of the API methods use Promises and the rest used callbacks. My bad. I'll make the API on the native side use Promises.